charts/mirrorbits-parent/templates/_helpers.tpl

```diff
@@ -30,3 +30,10 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
```

Comment:
This comment should be adapted (copied from L15 I suppose)

Reply:
https://github.com/jenkins-infra/helm-charts/pull/775